packages/wallet-service/tests/commons.test.ts

```diff
@@ -1169,3 +1171,20 @@ describe('getWalletBalancesForTx', () => {
     });
   });
 });
+
+test('getTokenListFromInputsAndOutputs', () => {
```

Comment:
Why is this test being added in this PR?

Reply:
To increase coverage since the global coverage was failing